### PR TITLE
a bit of editing on the Howto-Singularlity-run document

### PR DESCRIPTION
**Pull request by vpbrendel**

### CODE REVIEW

File: doc/06.2-Howto-Singularity-run.md [hunk 1]
```diff
@@ -5,57 +5,56 @@
...
-Since they do not run an entire operating system, containers are trivial to spin up and they share resources with other processes running on the same machine.
-Furthermore, they are much smaller in size, therefore much easier to distribute.
+Containerized applications run in isolated environments within the host operating system, sharing resources with other processes running on the same machine.
+Because they do not run an entire operating system, containers are trivial to spin up; and as they are much smaller in size, they are much easier to distribute.
 Finally, it is noteworthy that containers can also run on VMs.
 
 In summary, containers provide portability and reproducibility without significant overhead while VMs can provide compute infrastructure, but with significant overhead.  
-When we want to distribute our workflows to predictably run on arbitrary compute platforms, containers are more suitable than virtual machines.
+For those reasons, When we want to distribute our workflows to predictably run on a large number of distinct compute platforms, containers are more suitable than VMs.
```
Comment (littleblackfish) [inline doc/06.2-Howto-Singularity-run.md]: Unnecessary capitalization on When.

File: doc/06.2-Howto-Singularity-run.md [hunk 2]
```diff
@@ -116,38 +115,39 @@ When we provide such relative paths, we are implicitly referring to the working
...
 If `file.bam` was in a scratch space that is outside our home (`/scratch`), the command would look like this :
 
 ```
 singularity exec --bind /scratch bwasp.simg samtools view -h /scratch/file.bam  -o /scratch/file.sam
 ```
 
-And we are good to go, all we needed to add was really `--bind /scratch`.
-We also converted to full paths, but this may or may not be necessary depending on the exact set up.
+All we needed to add was `--bind /scratch`.
+We also used to full paths to indicate files, but this may or may not be necessary depending on the exact set up.
```
Comment (littleblackfish) [inline doc/06.2-Howto-Singularity-run.md]: 'to' is unnecessary in 'used to'
